src/schemas/components/SchemaForm.vue

```diff
@@ -1,32 +1,42 @@
 <template>
-  <p-form class="schema-form">
+  <p-form class="schema-form" @submit="submit">
     <template v-if="schema.properties">
-      <SchemaFormProperties v-model:values="values" :parent="schema" :properties="schema.properties" />
+      <SchemaFormProperties v-model:values="values" :parent="schema" :properties="schema.properties" :errors="errors" />
     </template>
   </p-form>
 </template>
 
 <script lang="ts" setup>
-  import { provide, computed } from 'vue'
+  import { provide, computed, ref } from 'vue'
+  import { useWorkspaceApi } from '@/compositions'
   import SchemaFormProperties from '@/schemas/components/SchemaFormProperties.vue'
   import { schemaInjectionKey } from '@/schemas/compositions/useSchema'
   import { schemaFormKindsInjectionKey } from '@/schemas/compositions/useSchemaFormKinds'
   import { Schema } from '@/schemas/types/schema'
   import { PrefectKind, SchemaValues } from '@/schemas/types/schemaValues'
+  import { SchemaValueError } from '@/schemas/types/schemaValuesValidationResponse'
 
-  const props = defineProps<{
+  const props = withDefaults(defineProps<{
     schema: Schema,
     values: SchemaValues,
     kinds: PrefectKind[],
-  }>()
+    loading?: boolean | null,
+  }>(), {
+    loading: null,
```

Comment:
without this `loading` is automatically defined as `false` but I won't this to be truly optional.